Examples/Counter/Parser.go

```diff
@@ -6,15 +6,18 @@ import (
 	"time"
 )
 
-type Export struct {
-	Field1 int `type:"counter" metric:"Field1"`
-}
-
 func main() {
-	var ht Export
+	var addToHandler prometheus_handler.HandlerStructure
 	for i := 0; i < 10; i++ {
-		ht.Field1 = i
-		result := prometheus_handler.GenericPromDataParser(ht)
+		addToHandler = append(addToHandler, struct {
```

Comment:
Better to also update the READMEs' of all the Examples since it'll explain the output according to the new format

Reply:
i'll make the changes